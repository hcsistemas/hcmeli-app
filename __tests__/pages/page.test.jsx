import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import Home from '@/app/page'

describe('Home', () => {
  it('renders page unchanged', () => {
    const { container } = render(<Home/>);

    expect(container)
      .toMatchSnapshot();
  });
})

