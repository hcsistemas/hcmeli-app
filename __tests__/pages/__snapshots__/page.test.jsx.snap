// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders page unchanged 1`] = `
<div>
  <main
    class="main"
  >
    <div
      class="search-bar-main"
    >
      <div
        class="search-bar-cont"
      >
        <div
          class="search-bar-cont-img"
        >
          <a
            href="/"
          >
            <img
              alt="meli-logo"
              class=""
              src="/meli.png"
              width="42px"
            />
          </a>
        </div>
        <div
          class="search-bar-cont-input"
        >
          <form
            action="/items"
            method="GET"
          >
            <div
              class="input-group mb-3"
            >
              <input
                class="form-control"
                name="search"
                placeholder="Nunca dejes de buscar"
                type="text"
                value=""
              />
              <div
                class="input-group-append"
              >
                <button
                  class="btn btn-secondary btn-bar"
                  type="submit"
                >
                  <i
                    class="bi bi-search"
                  />
                </button>
              </div>
            </div>
          </form>
        </div>
      </div>
    </div>
  </main>
</div>
`;
